src/wanna/core/deployment/models.py

```diff
@@ -92,6 +92,7 @@ class JobResource(GCPResource, Generic[JOB]):
     tensorboard: Optional[str]
     network: str
     job_config: JOB
+    encryption_spec_key_name: Optional[str]
```

Comment:
I would expect this to be used somewhere in `vertex_jobs.py`

Reply:
I agree, it doesnt seem to be used anywhere?